cameo/strain_design/deterministic/linear_programming.py

```diff
@@ -426,3 +430,394 @@ def _repr_html_(self):
+        A list of metabolite ID's for the the metabolites that should potentially be added to the medium
+    n_knockouts : int
+        The maximum allowed number of knockouts
+    n_knockins : int
+        The maximum allowed number of knockins
+    n_medium : int
+        The maximum allowed number of medium additions
+    use_solution_pool : Bool
+        Set to True when using the Gurobi solver to identify multiple alternative solutions at once
+    """
+    def __init__(
+            self, model, target, knockout_reactions, knockin_reactions, medium_additions,
+            n_knockouts, n_knockin, n_medium, remove_blocked=True, use_solution_pool=False, **kwargs
+    ):
+        super(GrowthCouplingPotential, self).__init__(**kwargs)
+
+        # Check model for forced fluxes, which should not be present. The zero solution must be allowed.
+        for r in model.reactions:
+            if r.lower_bound > 0 or r.upper_bound < 0:
+                raise ValueError(
+                    "%s has a forced flux. Remove the reaction or change its bounds for the function to work." % r.id)
+
+        # If a cobra.Reaction is given as target, convert to its ID string.
+        if isinstance(target, cobra.core.Reaction):
+            target = target.id
+        self.target = target
+
+        if target not in knockout_reactions:
+            knockout_reactions.append(target)
+
+        self.original_model = model
+        self.biomass_id = [r for r in model.reactions if r.objective_coefficient != 0][0].id
+        self.model = model = model.copy()
+
+        # Add boundary reactions for these metabolites named "ADD_<metabolitename>" to the model, with bounds (-10,0).
+        medium_addition_reactions = []  # List of the addition reactions added to the model
+        for m in medium_additions:
+            met = model.metabolites.get_by_id(m)
+            r = model.add_boundary(met, type="medium addition", reaction_id=("ADD_" + m), lb=-10, ub=0)
+            # r.type = "medium"
+            medium_addition_reactions.append(r.id)
+
+        # Set the solver to Gurobi for the fastest result. Set to CPLEX if Gurobi is not available.
+        if "gurobi" in cobra.util.solver.solvers.keys():
+            logger.info("Changing solver to Gurobi and tweaking some parameters.")
+            if "gurobi_interface" not in model.solver.interface.__name__:
+                model.solver = "gurobi"
+            # The tolerances are set to the minimum value. This gives maximum precision.
+            problem = model.solver.problem
+            problem.params.NodeMethod = 1  # primal simplex node relaxation
+            problem.params.FeasibilityTol = 1e-9
+            problem.params.OptimalityTol = 1e-3
+            problem.params.IntFeasTol = 1e-9
+            problem.params.MIPgapAbs = 1e-9
+            problem.params.MIPgap = 1e-9
+            # problem.params.TimeLimit = 200 # Use max 200 seconds when called, return best solution after that
```

Comment:
What happened to these parameters? If they are not important, can these lines be removed?